Sources/RInAppMessaging/Utilities/BundleInfo.swift

```diff
@@ -66,4 +74,12 @@ internal extension Bundle {
     private static var sdk: Bundle? {
         sdkBundle(name: "RInAppMessaging")
     }
+    
+    fileprivate func getRMCSdkVersion() -> String? {
+        guard let path = path(forResource: "RmcInfo", ofType: "plist"),
```

Comment:
is `RmcInfo.plist` present in the resources?

Reply:
It is present in RMC resources when IAM is integrated to RMC , the plist is fetched .